app.js

```diff
@@ -6,6 +6,7 @@ const path = require('path')
 const port = (process.env.PORT || 3000)
 const dto = require('directory-to-object')
 const yaml = require('js-yaml')
+const helperFunctions = require('./scripts/helper-functions.js')
```

Comment:
The `scripts` directory is currently used for deployment related scripts. I don't think it makes sense to put this file in there too. Suggest creating a `lib` directory and moving it there?